pkg/toolchains/git/BUILD.tmpl

```diff
@@ -0,0 +1,16 @@
+# This content is generated by {GENERATOR}
+load("@rules_pkg//toolchains/git:git.bzl", "git_toolchain")
+
+git_toolchain(
+    name = "git_auto",
+    # path to git executable
+    path = "{GIT_PATH}",
+    # path to the top of the git client (but really any folder under it)
+    client_top = "{GIT_ROOT}",
```

Comment:
I don't really hear people calling cloned repositories git "clients".  I had to think about this for a moment; it may be worth calling these "checkouts", or simply "workspaces".

I don't feel too strongly about this, though.